### DISCUSSION TITLE
2022-02-21 Core team update

### DISCUSSION THREAD

**Discussion by mm-near** [near/community#224] [Core]
# TL;DR

* more storage / rocksdb analysis & planned improvements
* preparing designs on how to stress-test nearcore (network, storage and chain code)
* improving debuggability of our system

# Done

@jc-near - Hello-world page for the sync debugging tool

@mzhangmzz - presentation on EthDenver
@mzhangmzz - First iteration of the stress testing client

@pompon0 - PoC of a chain synchronisation loadtest for network
@pompon0 - experimenting with how to replace Actix
@Longarithm - Replace Vec<u8> with Arc in trie cache - should bring 5% improvement

@EdvardD - Compression analysis for rocksdb

@mm-near - rocksdb CPU performance testing
@mm-near - loadtesting localnet (single node) 
@mm-near - working on the debug page to see the detailed performance of the recent blocks
